client/my-sites/plans/jetpack-plans/constants.ts

```diff
@@ -479,3 +485,12 @@ export const DOWNGRADEABLE_PLANS_FROM_PLAN: Record< string, string[] > = {
 		PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
 	],
 };
+
```

Comment:
We've got an updated version of the map with `LEGACY_TO_RECOMMENDED_MAP` in `client/my-sites/plans/jetpack-plans/plan-upgrade/constants.ts`

Reply:
I was going to delete that and then I totally forgot it. Thanks!